cmd/query.go

```diff
@@ -46,22 +47,16 @@ func queryClientCmd(ctx *config.Context) *cobra.Command {
 			}
 			c := chains[args[1]]
 
-			height, err := cmd.Flags().GetInt64(flags.FlagHeight)
+			height, err := c.LatestHeight()
```

Comment:
Was the original code that referenced the `height` flag wrong?

Reply:
No, I've reverted the change and implemented the same option for other queriers(connection, channel) at https://github.com/hyperledger-labs/yui-relayer/pull/83/commits/64f72896878ee712ec823c27d77181b0ed08e27f